### PR TITLE
fixing a 16:9 aspect ratio

### PR DESCRIPTION
**Pull request by norgeous**

### CODE REVIEW

**Review by ScottyXIII — COMMENTED** [review]
image

I should prob update my version of node. Should I go latest or just `^18.0.0 || >=20.0.0`?

File: src/scenes/game-scene.ts [hunk 1]
```diff
@@ -97,7 +97,7 @@ class GameScene extends Phaser.Scene {
     this.level = new Level(this, levelConfig);
     this.audio = new Audio(this, soundConfig);
 
-    this.audio?.playAudio('music2');
+    this.audio.playAudio('music2');
```
Comment (ScottyXIII) [inline src/scenes/game-scene.ts:100]: Does this not complain about this.audio potentially being undefined?

**Review by ScottyXIII — APPROVED** [review]

File: src/scenes/game-scene.ts (hunk 1 above)
Comment (norgeous) [reply src/scenes/game-scene.ts:100]: `this.audio` defined in the line above this one!
so no, it doesn't complain...

File: src/config.ts [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+// 16:9 horizontal sizes
+// - 360p = 640✕360
+// - 480p = 854✕480
+// - 720p = 1280×720
+// - 1080p = 1920✕1080
+
+const config: Phaser.Types.Core.GameConfig = {
+  type: Phaser.WEBGL,
+  width: 1280,
+  height: 720,
```
Comment (norgeous) [inline src/config.ts:13]: 720p

**Pull request merged**